src/syscall_hook.c

```diff
@@ -188,7 +153,7 @@ ssize_t recv(int sockfd, void *buf, size_t len, int flags)
 ssize_t write(int fd, const void *buf, size_t count)
 {
     ssize_t n;
-
+    
```

Comment:
Don't do this!

Reply:
I didn't understand the meaning of minimizing, is there any detailed explnation?

Reply:
You should avoid unrelated changes such as adding/deleting a blank line.

Reply:
That is, every line you propose to change should be exactly functioned without cosmetic.

Reply:
I'll pay more attention to it in the future.